### ISSUE TITLE
Feature request: Option to disable translations for specific pages

### ISSUE THREAD

**Issue by bummey** [vinissimus/next-translate#565]
Hey there, thanks a lot for this library! Been using it a while, but this is the first time I stumbled upon this issue. For some reason when next-translate tries to compile my api route, it just breaks completely:
```
event - build page: /api
🚨 [next-translate] In Next 10.x.x there is an issue related to i18n and getInitialProps. We recommend to replace getInitialProps to getServerSideProps on /api.ts. Issue: https://github.com/vercel/next.js/issues/18396
API resolved without sending a response for /api, this may result in stalled requests.
```
When I disable the plugin in next.config.js, everything works just fine. That's why I request an option to exclude specific pages from next-translate, since for my specific use-case I don't even need it in my api routes.

Thanks again and I hope you'll consider it for a future release! :-)

**Comment by aralroca** [comment]
everything under `/api` (as a folder) should not compile... Like `/api/example.ts`, etc. 

The `api.ts` file is understood as a page to compile... Maybe the simplest solution is to add this use case. It is only contemplated that `api` as a folder...

**Comment by aralroca** [comment]
@bummey I prereleased the fix under [1.0.6-canary.2](https://github.com/vinissimus/next-translate/releases/tag/1.0.6-canary.2). Please confirm if it works for you now! Thanks!

**Comment by bummey** [comment]
Works like a charm, thanks a lot!
